packages/firebase_data_connect/firebase_data_connect/lib/src/common/dataconnect_error.dart

```diff
@@ -28,6 +28,42 @@ class DataConnectError extends FirebaseException {
   final DataConnectErrorCode dataConnectErrorCode;
 }
 
+/// Error thrown when an operation is partially successful.
+class DataConnectOperationError<T> extends DataConnectError {
+  DataConnectOperationError(
+      DataConnectErrorCode code, String message, this.response)
+      : super(code, message);
+  final DataConnectOperationFailureResponse<T> response;
+}
+
+/// Nested class containing errors and decoded data.
+class DataConnectOperationFailureResponse<T> {
+  DataConnectOperationFailureResponse(this.errors, this.data, this.decodedData);
+  final Map<String, dynamic>? data;
```

Comment:
I think we have to call this `rawData` per API review feedback. 

cc: @dconeybe

Reply:
Yes, that is true: `data`/`decodedData` has been renamed to `rawData`/`data`, as per api council feedback.

Reply:
Updated